### PR TITLE
Add proxy enhancements 

### LINKED ISSUE #2553: UI Enhancement for Add proxy view
image

1. Add examples in texts (italic and appearing disable) to fields: (can be the same style as that of HTTP API URL
    - a. Host
    - b. Port
2. Rename HTTP API URL as **Configuration API Endpoint**
3. Give Infor Tip explaining what is TLS. The message should contain the full meaning of TLS.

### PR DESCRIPTION
**Pull request by 55**
Closes #2553 

- [x] Add examples in texts (italic and appearing disable) to fields: (can be the same style as that of HTTP API URL
  - [x] a. Host
  - [x] b. Port
- [x] Rename HTTP API URL as Configuration API Endpoint
- [x] Give Info Tip explaining what is TLS. The message should contain the full meaning of TLS.

### CODE REVIEW

File: core/lib/i18n/en.i18n.json [hunk 1]
```diff
@@ -820,21 +820,24 @@
...
                     },
                     "host": {
-                      "label": "Host"
+                      "label": "Host",
+                      "placeholder": "https:\/\/example.com\/"
                     },
                     "port": {
-                      "label": "Port"
+                      "label": "Port",
+                      "placeholder": "i.e. 224"
```
Comment (matleppa) [inline core/lib/i18n/en.i18n.json]: E.g.  instead of i.e.

File: core/lib/i18n/en.i18n.json [hunk 2]
```diff
@@ -820,21 +820,24 @@
...
                     },
                     "tls": {
                       "label": "TLS"
```
Comment (matleppa) [inline core/lib/i18n/en.i18n.json]: Should there be a tooltip explaining the meaning og TLS?

Comment (55) [inline core/lib/i18n/en.i18n.json]: Yep, in a process of figuring out the best way.

**Comment by 55** [comment]
@Nazarah are there any other ways to explain the purpose of TLS but tooltip helper?
We have limitations here, since we are generating these fields all at once and not separately one by one.
Perhaps, transcript of it in brackets will work?

@brylie @marla-singer any thoughts on this?

**Comment by marla-singer** [comment]
@55 I suggest using "title" attribute for that. What do you think?

**Comment by 55** [comment]
@marla-singer yes, I also think that this might be an overdo with help here.

**Comment by Nazarah** [comment]
Ok with the changes. Good work Ilya!

**Pull request merged**